кортежи.py

```diff
@@ -0,0 +1,19 @@
+# Создайте список ['a', 'b', 'c'] и сделайте из него кортеж.
+lst1 = [el for el in 'abc']
+kort1 = tuple(lst1)
+print(kort1)
+
+# Создайте кортеж ('a', 'b', 'c'), И сделайте из него список
+kort2 = (el for el in 'abc')
+lst2 = list(kort2)
+print(lst2)
+# Сделайте следующие присвоения одной строкой a = 'a', b=2, c=’python’.
+a, b, c = 'a', 2, 'python'
+
+# Создайте кортеж из одного элемента, чтобы при итерировании по этому
+# элементы последовательно выводились значения 1, 2, 3. Убедитесь что
+# len() исходного кортежа возвращает 1.
+lst3 = (123,)

```

Comment:
it's not tuple with iterable first element

Reply:
rdy